sources/europe/es/geoEuskadi-Karto.geojson

```diff
@@ -7,9 +7,9 @@
             "required": true
         },
         "name": "geoEuskadiren Kartografia",
-        "url": "https://www.geo.euskadi.eus/geoeuskadi/rest/services/U11/WMTS_KARTO/MapServer/WMTS/1.0.0/WMTSCapabilities.xml",
+        "url": "https://www.geo.euskadi.eus/geoeuskadi/rest/services/U11/WMTS_KARTO/MapServer/WMTS/tile/1.0.0/U11_WMTS_KARTO/default/GoogleMapsCompatible/{z}/{y}/{x}.png",
```

Comment:
```suggestion
        "url": "https://www.geo.euskadi.eus/geoeuskadi/rest/services/U11/WMTS_KARTO/MapServer/WMTS/tile/1.0.0/U11_WMTS_KARTO/default/GoogleMapsCompatible/{zoom}/{y}/{x}.png",
```

per the CI error

> ERROR:root:Error in sources/europe/es/geoEuskadi-Karto.geojson : {z} found instead of {zoom} in tile url

and the spec

> https://github.com/osmlab/editor-layer-index/blob/gh-pages/CONTRIBUTING.md#source-url